util/network/client.py

```diff
@@ -48,6 +48,10 @@ def send_data(str_data):
     def is_connected():
         return ClientUtils.is_connected()
 
+    @staticmethod
+    def get_sid():
+        return sio.get_sid()
+

```

Comment:
to compare with the other sid, and select who starts sending data, who starts receiving